script/openqa-clone-custom-git-refspec

```diff
@@ -124,10 +124,11 @@ Please try 'curl $json_url' or select another job, e.g. in the same scenario: $h
         local old_casedir
         old_casedir=$(echo "$json_data" | jq -r '.CASEDIR') || throw_json_error "$json_url" "$json_data"
         if [[ ${old_productdir:0:1} == "/" ]]; then
```

Comment:
It would be useful to add a comment explaining *why* we need to set the product dir here.

Reply:
ok. I added a comment "set a proper productdir pointing to the checked out repository for absolute+relative paths"